ui/src/app/pages/homepage/quick-tour-modal.tsx

```diff
@@ -325,7 +337,7 @@ export class QuickTourReact extends React.Component<QuickTourReactProps, QuickTo
           <div style={styles.breadcrumbs}>
             {panels.map((p, i) => {
               return <React.Fragment key={i}>
-                <div style={{width: '128px'}}>
+                <div style={{width: i === panels.length - 1 ? null : '128px'}}>
```

Comment:
I really dislike this area of code, but it blows up the scope to try to fix it, because the container magic would get complicated.